src/MetricsProvider.ts

```diff
@@ -1,4 +1,4 @@
-import { COUNTLY_API_URL } from './config.js'
+import { COUNTLY_API_URL } from './config'
```

Comment:
If the published output is ESM, you need the extension for loading individual files, and you can't import from a directory without `/index.js`.  This different to outputting CJS which will work without extensions etc as before.

Tediously this breakage only shows up at runtime which I think is what @SgtPooki has discovered testing this with companion.